multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -217,8 +223,10 @@ def __init__(
         self._warn_if_exist = warn_if_exist
         self._enable_progress_bar = enable_progress_bar if enable_progress_bar is not None else True
 
-        if problem_type is not None and problem_type.lower() == ZERO_SHOT:
-            self._config, self._model, self._data_processors = init_zero_shot(hyperparameters=hyperparameters)
+        if pipeline is not None:
```

Comment:
Should we raise a DeprecatedError about `problem_type == "zero_shot"`? Since it's in 0.5.1.

Reply:
I set the pipeline=ZERO_SHOT_IMAGE_CLASSIFICATION if we see problem_type == "zero_shot" and added a deprecated warning.